playground/mocks/data/idp/idx/error-authenticator-verification-passcode-change-on-prem.json

```diff
@@ -7,9 +7,12 @@
     "type": "array",
     "value": [
       {
-        "message": "Wait for token to change, then enter the new tokencode.",
+        "message": "Pin accepted, Wait for token to change, then enter new passcode",
         "i18n": {
-          "key": "errors.E0000106"
+          "key": "errors.E0000113",
```

Comment:
why did we change it from 106 to 113?

Reply:
106 is translated to the next token mode error. 113 is the one that maps to `{0}`. This has only been changed for the verify flow